proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (C) 2018 Velocity Contributors
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+velocity.error.already-connected=Jesteś już połączony z tym serwerem!
+velocity.error.already-connected-proxy=Jesteś już połączony z tym proxy!
+velocity.error.already-connecting=Już próbujesz połączyć się z serwerem!
+velocity.error.cant-connect=Nie udało się połączyć z {0}: {1}
+velocity.error.connecting-server-error=Nie udało ci się się połączyć z {0}. Spróbuj ponownie później.
+velocity.error.connected-server-error=Twoje połączenie z {0} napotkało problem.
+velocity.error.internal-server-connection-error=Wystąpił wewnętrzny błąd połączenia z serwerem.
+velocity.error.logging-in-too-fast=Logujesz się zbyt szybko, spróbuj ponownie później.
+velocity.error.online-mode-only=Na serwer mogą wejść osoby tylko z oryginalną kopią gry. Jeśli jesteś zalogowany na swoim koncie Minecraft, spróbuj zrestartować gre.
+velocity.error.player-connection-error=Wystąpił błąd wewnętrzny w twojim połączeniu.
+velocity.error.modern-forwarding-needs-new-client=Ten serwer jest kompatybilny tylko z Minecraft 1.13 i nowszymi wersjami.
+velocity.error.modern-forwarding-failed=Twój serwer nie wysłał żądania przekazania do serwera proxy. Upewnij się, że serwer jest skonfigurowany do przesyłania Velocity.
+velocity.error.moved-to-new-server=Zostałeś wyrzucony z {0}: {1}
+velocity.error.no-available-servers=Brak dostępnych serwerów z którymi można by się połączyć. Spróbuj ponownie później lub skontaktuj się z administratorem.
+
+# Commands
+velocity.command.generic-error=Wystąpił błąd podczas uruchamiania tego polecenia. 
+velocity.command.command-does-not-exist=To polecenie nie istnieje.
+
+velocity.command.players-only=Tylko gracze mogą uruchomić to polecenie.
+velocity.command.server-does-not-exist=Podany serwer {0} nie istnieje.
+
+velocity.command.server-current-server=Jesteś obecnie połączony z {0}.
+velocity.command.server-too-many=Skonfigurowano zbyt wiele serwerów. Użyj uzupełniania kart, aby wyświetlić wszystkie dostępne serwery.
```

Comment:
For sure we are not completing any cards here. I'd change the beginning of second sentence to either of these:

- `Użyj uzupełniania tabem` (might be good to accent TAB by writing `TABem`)
- `Użyj uzupełniania klawiszem tab`

Reply:
Maybe `Użyj uzupełniania tabulatorem`?